Pages documentation (#598)
sharetribe/flex-docs: src/docs/concepts-development/content-management/index.md

All sections have a title, an ingress, and a button for a call to
action.

### Article

Article sections are great for prose text. If your content is longer,
you can add blocks with different levels of headings and text
paragraphs. The blocks are displayed in a column one after the other, as
if on a blog page.

### Columns

Columns sections divide the blocks within the section into pre-defined
columns, and the operator can decide the number of columns between 1-4.
The columns pattern is great for featuring specific options in parallel,
whether they are benefits, testimonials, or featured locations.

### Features

Features sections, by default, show blocks of visual and text content so
that the block contents are on a single row, alternating the order, and
different blocks are in a column. As with all content sections, the
designer of your website can make changes as to how exactly the blocks
get displayed on the page.

## Blocks

Currently, Flex has a single type of block that you can add to any
section. The block contains a title, and you can determine the level of
the title (page title, section title, section subtitle). A block can
also have text content and a button.

> Block text content can be modified with Markdown. This allows the

Reviewer: Should we have some examples of how markdown is edited or should that be in operator-guides

Author: I think we could add a link to Markdown documentation on this, and then have more examples in the operator guide, ping @thomas-rocca .

Author: Thanks! Link added to article.

---
title: Content management in Flex
slug: content-management
updated: 2022-08-02
category: concepts-development
ingress:
  This article introduces the Flex content management system and how
  static pages can be managed in Flex Console
published: true
---

In 2022-09, Flex introduces a headless CMS-like feature to create static
pages in Flex. Initially, the feature allows operators to modify their
landing pages without code. Going forward, it will be possible to create
new pages as well.

## Flex Content management glossary

> _TODO: A better place for this section?_

Reviewer: If we have these as subheadings we can link to them from other articles

---
title: Content management in Flex
slug: content-management
updated: 2022-08-02
category: concepts-development
ingress:
  This article introduces the Flex content management system and how
  static pages can be managed in Flex Console
published: true
---

> In 2022-09, Flex introduces a headless CMS-like feature to create static

Author: This article has a draft being edited in https://docs.google.com/document/d/1Th6Eu4lhaIyplODU5qgpSmqApf48ZssceyCCzWyMqYE/edit?usp=sharing – the goal for this article is that it should be accessible to both technical and non-technical audiences. We need to decide if there's another concepts-type article that needs to happen for the technical audience, or if this is enough.

---
title: Content management in Flex
slug: content-management

> updated: 2023-01-11

Author: Fix updated dates across the PR

category: concepts-development
ingress:
  This article introduces the Flex content management system and how

>   static pages can be managed in Flex Console

Reviewer: Comment:

Come to think of it, I'm not sure if _static_ is the term we want to use to describe pages dynamically managed through Pages. "A CMS allows you to build static content pages" is a bit of an oxymoron.
 Should we instead use "content page" instead of "static page"?
 
 @SariSaar

published: true
---

> In 2023-02, Flex introduces a headless CMS-like feature to create static

Reviewer: ```suggestion
In 2023-02, Flex introduces a lightweight headless CMS to create static
```

Author: Fixed!

> pages in Flex. The feature allows operators to modify a selection of

Reviewer: ```suggestion
pages in Flex. The feature allows operators to modify four
```

- FTW-product v.X.X.X

For earlier template versions, you can either pull upstream updates or
make the changes manually. You can check this article on
[static content rendering in FTW](/ftw/page-builder/) for context, and
these PRs for the necessary changes:

// TODO update repo links

- [FTW-daily]()
- [FTW-hourly]()
- [FTW-product]()

To see your page changes in your client application, you need to save
your changes in Flex Console and navigate to the corresponding page in
your app.

Alternatively, if you have an application running in the address set as
your **Marketplace URL**, you can click the "View page" link next to the
page title in Flex Console.

![View page](./view-page.png)

This will open the page in your **Marketplace URL** address.

### Publishing pages from test to production

When you first take your markeptlace to production, your production
marketplace is created as a duplicate of your test marketplace. This
includes all your page assets.

You can also modify your marketplace content pages after going to
production. You will still need to make and test the changes in your
test environment. Then, after you are happy with all your page changes,

> you can click the "Copy to production" button

Reviewer: ```suggestion
you can click the "Copy to production" button.
```

---
title: Content management in Flex
slug: content-management
updated: 2022-07-28
category: concepts-development
ingress:
  This article introduces the Flex content management system and how
  static pages can be managed in Flex Console
published: true
---

In 2022-09, Flex introduces a headless CMS-like feature to create static
pages in Flex. Initially, the feature allows operators to modify their
landing pages without code. Going forward, it will be possible to create
new pages as well.

## Content modeling in Flex

> 

Author: TODO: Glossary of terms

Reviewer: Could be:

Page Builder
Content modelling
Content management
Page schema
Assets
Asset Delivery API
Page
Sections
Blocks
Fields
Page asset data
